python/pyarrow/includes/libarrow.pxd

```diff
@@ -2334,6 +2338,7 @@ cdef extern from "arrow/compute/api.h" namespace "arrow::compute" nogil:
 
         DatumType kind() const
         c_string ToString() const
+        c_string ToString(DatumType kind)
```

Comment:
Hmm, this is not a `Datum` method, you should move this out of the `cppclass` declaration.

Reply:
Yes, you're right. I moved it.